app-utils/7-zip/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=7-zip
+PKGSEC=utils
+PKGDEP="gcc-runtime glibc"
+BUILDDEP="llvm"
+PKGDES="7-Zip file archiver supporting a range of compression formats"
```

Comment:
A file archive manager with support for multiple formats

Reply:
Fixed.